decoders/connector/dragino/cpl03-lb

```diff
@@ -0,0 +1,241 @@
+/* eslint-disable default-case */
```

Comment:
You need to follow the order of the files that is defined in the manifest, following this file structure.
![Screenshot 2024-07-08 at 10 04 28](https://github.com/tago-io/decoders/assets/14567480/eb751596-b362-434b-aa1f-353e933f0c60)


Read the [README.md](https://github.com/tago-io/decoders/blob/main/README.md) to better understand and guide you.